tests/e2e/velero_yaml_suite_test.go

```diff
@@ -0,0 +1,42 @@
+package e2e
+
+import (
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+type dpaTemplateParams struct {
+	DpaName         string
+	BslRegion       string
+	Provider        string
+	CredentialsName string
+	BucketName      string
+	Prefix          string
+}
+
+var _ = AfterEach(func() {
+	// Delete Velero CR once the test has finished
+	err := vel.Delete()
+	Expect(err).ToNot(HaveOccurred())
+})
+
+var _ = Describe("Test Velero CR creation with", func() {
+	It("One Backup Storage Location templated from yaml", func() {
+		// Create Velero CR and verify it is successful
+		veleroTemplate, _ := filepath.Abs("templates/velero_bsl_template.tmpl")
```

Comment:
NIT: template isn't a BSL, name should be:
```
templates/dpa_template.tmpl
```

Reply:
> NIT: template isn't a BSL, name should be:
> 
> ```
> templates/dpa_template.tmpl
> ```

Sorry, missed that while addressing last review comments. Will update